traffic_ops/app/lib/UI/DeliveryService.pm

```diff
@@ -921,6 +919,10 @@ sub update {
 			}
 		}
 
+		my $new_hostname = UI::SslKeys::get_hostname($self, $id, $update);
+		$upd_ssl = 1 if $old_hostname ne $new_hostname;
```

Comment:
good catch! I think the code change is missing when doing merge.

The original change is made in TC 1.7. When picked up to master, looks like the code is lost :-(